13_adult.do

```diff
@@ -32,18 +59,23 @@ capture confirm variable sh246s sh255s sh264s sh246d sh255d sh264d
 	}
 	
 *a_bp_dial	18y+ diastolic blood pressure (mmHg) in adult population 
-
+	capture confirm variable sh315b sh324b sh334b
+    if _rc==0 {
+		ren (sh315b sh324b sh334b) (sh246d sh255d sh264d) 	
+    }
+	
 	capture confirm variable sh246d sh255d sh264d
 	if _rc == 0 {
 	egen a_bp_dial = rowmean(sh246d sh255d sh264d)
 	}
 	if _rc != 0{
 	gen a_bp_dial = .
 	}
+
 	
 *a_hi_bp140_or_on_med	18y+ with high blood pressure or on treatment for high blood pressure	
 	gen a_hi_bp140=.
-    replace a_hi_bp140=1 if a_bp_sys>=140 | a_bp_dial>=90 
+    replace a_hi_bp140=1 if (a_bp_sys>=140 & a_bp_sys!=.) | (a_bp_dial>=90 & a_bp_dial!=.)

```

Comment:
Thanks a lot, this is very helpful, I will review the other rounds to make the changes accordingly.